src/styles/Base/theme.scss

```diff
@@ -0,0 +1,38 @@
+/* Resets */
+
+ul {
+
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+
+  text-decoration: none;
+  color: #fff;
+}
+
+
+/* Colors */
+
+$main-black: #1F1C17;
+
+$main-yellow: #FFA10C;
+
+$main-brown: #593C1A;
+
+$main-white: #FAFAFA;
+
+/* Paddings */
```

Comment:
I say remove the padding styles you've added for now until we actually decide on the vertical rythm of things I think its very difficult to tell from now